utility/BUILD.bazel

```diff
@@ -54,6 +54,7 @@ kt_android_library(
         ":resources",
         "//app:crashlytics",
         "//app:crashlytics_deps",
+        "//model:arguments_java_proto_lite",
```

Comment:
Why is this needed by //utility?

Reply:
As PolicyPage arguments proto is been used in HtmlParser i.e in Utility.

Reply:
In that case we should probably use an enum defined in utility when interacting with utility components. This is an app-module only proto so it's not something we should depend on in utility (which also makes sense conceptually: HtmlParser shouldn't have knowledge about the types of pages being shown to users).

Reply:
By enum defined in utility you mean PolicyPage has to be defined in utility instead of creating proto ?

Reply:
Created PolicyType enum for utilities and used PolicyPage protos for UI.